clean.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Why are we not doing it via the MAKE build tool as done in K8s-mixins project?

Reply:
It was a temporary thing. Using Makefile now.